docs/install/proxmox/proxmox-setup.md

```diff
@@ -0,0 +1,217 @@
+---
+title: Installing Colony with Promox
+description: Follow the steps to install Colony with Proxmox
+sidebar_position: 2
+---
+
+## Installing Colony with Proxmox
+
+This guide walks through the setup of Laptop-0, a machine for running Colony, and 3 additional VM's for Colony to auto discovery and install a K3S cluster.  Some items to note:
+
+- Colony requires Docker.
+- If you have a firewall on your host machine (laptop 0) it needs to allow the following ports (ports TBA).
```

Comment:
This was a half thought out idea, let's remove it.

Reply:
Feel free to just suggest an edit for what you think should be removed.